staging/src/k8s.io/mount-utils/mount_windows.go

```diff
@@ -300,13 +309,13 @@ func (mounter *SafeFormatAndMount) formatAndMountSensitive(source string, target
 		return err
 	}
 	driverPath := volumeIds[0]
-	target = NormalizeWindowsPath(target)
-	output, err := mounter.Exec.Command("cmd", "/c", "mklink", "/D", target, driverPath).CombinedOutput()
-	if err != nil {
-		klog.Errorf("mklink(%s, %s) failed: %v, output: %q", target, driverPath, err, string(output))
-		return err
+
+	// Mount the disk
+	klog.V(4).Infof("Attempting to mount disk %s in %s format at %s", driverPath, fstype, target)
+	if err := mounter.MountSensitive(driverPath, target, fstype, options, sensitiveOptions); err != nil {
+		return NewMountError(UnknownMountError, err.Error())
 	}
-	klog.V(2).Infof("formatAndMount disk(%s) fstype(%s) on(%s) with output(%s) successfully", driverPath, fstype, target, string(output))
+
```

Comment:
is it intended change of logging level from 2 to 4?

Reply:
Excluded change. Will revisit this in a later PR.